src/test/java/io/openliberty/tools/intellij/it/MacOSAllowPopupTest.java

```diff
@@ -0,0 +1,42 @@
+/*******************************************************************************
+ * Copyright (c) 2025 IBM Corporation.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.tools.intellij.it;
+
+import com.intellij.remoterobot.RemoteRobot;
+import com.automation.remarks.junit5.Video;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.api.condition.EnabledOnOs;
+import org.junit.jupiter.api.condition.OS;
+
+
+public class MacOSAllowPopupTest {
```

Comment:
Discussed with @turkeylurkey . We'd like to suggest renaming the class to "BaseOSUtilities". It seems strange to have our common test classes extending a Mac specific class. This class could then be added to in the future if other "tests" or setup becomes necessary for any OS.